components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/css/console.css

```diff
@@ -137,3 +151,12 @@
     white-space: pre-wrap;
 }
 
+.console-container .close {
+    font-size: 10px;
+}
+
+.console-container .close i {
+    color: #ccc;
+    line-height: 19px;
+    padding-right: 10px;
```

Comment:
Can't we use `%` instead of `px`?

Reply:
Here also we have used fixed values, no requirement of relative values.